src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -319,6 +320,28 @@ void DoRemoveAllFabrics(intptr_t)
 {
     OpenDefaultPairingWindow(ResetAdmins::kYes);
 }
+
+CHIP_ERROR SendNOCResponse(chip::app::Command * commandObj, EmberAfNOCResponseStatus status, uint8_t index, ByteSpan debug_text)
+{
+    app::CommandPathParams cmdParams = { emberAfCurrentEndpoint(), /* group id */ 0, ZCL_OPERATIONAL_CREDENTIALS_CLUSTER_ID,
+                                         ZCL_NOC_RESPONSE_COMMAND_ID, (chip::app::CommandPathFlags::kEndpointIdValid) };
+    TLV::TLVWriter * writer          = nullptr;
+
+    VerifyOrReturnError(commandObj != nullptr, CHIP_ERROR_INCORRECT_STATE);
+
+    ReturnErrorOnFailure(commandObj->PrepareCommand(cmdParams));
+    writer = commandObj->GetCommandDataElementTLVWriter();
+    ReturnErrorOnFailure(writer->Put(TLV::ContextTag(0), status));
+    if (status == EMBER_ZCL_NOC_RESPONSE_STATUS_SUCCESS)
+    {
+        ReturnErrorOnFailure(writer->Put(TLV::ContextTag(1), index));
+    }
+    ReturnErrorOnFailure(writer->Put(TLV::ContextTag(2), debug_text));
```

Comment:
Please make sure there's a TODO to fix this to be a string once we support strings in command fields?